docs/proposals/202106120-edgex-integration.md

```diff
@@ -80,25 +80,235 @@ As the deployment topology diagram shows.  Each components in the EdgeX instance
 
 Following is the definition of `EdgeX` CRD and its related Golang struct. The CRD will be created when the EdgeX controller is up and running. 
 ```yaml
-apiVersion: apiextensions.k8s.io/v1beta1
+---
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    ...
+    controller-gen.kubebuilder.io/version: v0.4.1
   creationTimestamp: null
-  name: edgex.apps.openyurt.io
+  name: edgexs.device.openyurt.io
 spec:
-  ...
-  optionalComponents:
-    description: 
-    type: object
+  group: device.openyurt.io
+  names:
+    kind: EdgeX
+    listKind: EdgeXList
+    plural: edgexs
+    singular: edgex
+  scope: Cluster
```

Comment:
I think the scope should be `Namespaced`, so keep consistent with `UnitedDeployment`